src/SourceLink.Common/build/Microsoft.SourceLink.Common.props

```diff
@@ -7,14 +7,13 @@
 
   <PropertyGroup>
     <!--
-      Do not query source control manager for any information when building in the IDE or for Live Unit Testing.
-      Features that need this information will be disabled if false.
+      Used to suppress querying source control and features that use the information (e.g. git commit SHA).
     -->
-    <EnableSourceControlManagerQueries Condition="'$(EnableSourceControlManagerQueries)' == '' and '$(DesignTimeBuild)' != 'true' and '$(BuildingForLiveUnitTesting)' != 'true'">true</EnableSourceControlManagerQueries>
-    
+    <EnableSourceControlManagerQueries Condition="'$(EnableSourceControlManagerQueries)' == ''">true</EnableSourceControlManagerQueries>
+
     <!--
       Do not generate SourceLink when building in the IDE or for Live Unit Testing.
     -->
     <EnableSourceLink Condition="'$(EnableSourceLink)' == '' and '$(DesignTimeBuild)' != 'true' and '$(BuildingForLiveUnitTesting)' != 'true'">true</EnableSourceLink>
```

Comment:
@rainersigwald I kept this suppress in DTB since this only affects generated SourceLink.json that only impacts emit, so it should not be needed in DTB. Makes sense? Or should we enable it as well to have 100% consistency between DTB and command line build?

Reply:
I think this makes sense to me.

Reply:
Is the SourceLink.json passed in as a compiler input? E.g. as an addtional item?

Reply:
It's a separate command line switch `/sourcelink`.